### DISCUSSION TITLE
Sub Pages in Quarto Dashboard

### DISCUSSION THREAD

**Discussion by FGew** [quarto-dev/quarto-cli#7504] [Feature Requests]
### Description

Flexdashboard allowed for grouping of pages so that the user clicks the page header and then a dropdown would appear so the user could click a subpage. Can we have this in Quarto Dashboard?

 ##dashboard

**Comment by FGew** [comment]
In flexdashboard, subpages were specified like this:

````

Subpage Name {data-navmenu="Group1"}
=====================================  

````

**Reply by mcanouil** [reply]
dropdown menu in navbar is planned.

**Reply by rollingstone09** [reply]
+1 pretty please
